keep-ui/app/workflows/[workflow_id]/side-nav-bar.tsx

```diff
@@ -28,7 +28,7 @@ export default function SideNavBar({ workflow }: { workflow: Workflow }) {
     ];
 
     return (
-        <div className="flex flex-col gap-10 pt-6 top-20 p-1 max-w-[270px]">
```

Comment:
why?

Reply:
In this issue context, it is irrelevant but This style change gives us more responsiveness than before.

Note: I only created this component. I have tested it.